threedigrid_builder/constants.py

```diff
@@ -120,6 +120,8 @@ class BoundaryType(IntEnum):
     SOMMERFELD = 5
     GROUNDWATERLEVEL = 6
     GROUNDWATERDISCHARGE = 7
+    TOTALDISCHARGE2D = 8
+    GROUNDWATERTOTALDISCHARGE2D = 9
```

Comment:
I would be in favor of `TOTAL_DISCHARGE_2D`, and `GROUNDWATER_TOTAL_DISCHARGE_2D`

Reply:
I am also in favor of using underlines to separate the words for better readability, but I just followed the pattern in the existing names